cmd/earthly/buildkit.go

```diff
@@ -206,11 +207,12 @@ func (app *earthlyApp) isUsingSatellite(cliCtx *cli.Context) bool {
 	return app.cfg.Satellite.Name != "" || app.satelliteName != ""
 }
 
-func (app *earthlyApp) reserveSatellite(ctx context.Context, cloudClient cloud.Client, name, orgID string) error {
+func (app *earthlyApp) reserveSatellite(ctx context.Context, cloudClient cloud.Client, name, orgID, gitAuthor string) error {
 	console := app.console.WithPrefix("satellite")
 	out := make(chan string)
 	var reserveErr error
-	go func() { reserveErr = cloudClient.ReserveSatellite(ctx, name, orgID, out) }()
+	isCI := false
```

Comment:
@alexcb @vladaionescu Will this reflect `--ci` or something else?

Reply:
Something else - it's going to be this heuristic: https://github.com/earthly/earthly/blob/main/analytics/analytics.go#L25.

Reply:
Oh, nice. Got it.